api/build.gradle

```diff
@@ -216,8 +218,8 @@ dependencies {
   }
   compile 'com.google.code.gson:gson:+'
   compile 'com.google.oauth-client:google-oauth-client-jetty:1.23.0'
-  compile 'com.squareup.okhttp:okhttp:2.6.0'
-  compile 'com.squareup.okhttp:logging-interceptor:2.6.0'
+  compile "com.squareup.okhttp:okhttp:${OKHTTP_VERSION}"
+  compile "com.squareup.okhttp:logging-interceptor:${OKHTTP_VERSION}"
```

Comment:
nit: single quotes for consistency

Reply:
Intentional - in Groovy you cannot interpolate within a single-quoted string.

http://docs.groovy-lang.org/latest/html/documentation/#_single_quoted_string

Reply:
Thank you - yes, I had forgotten that, sorry for the red-herring.